### PR TITLE
Misc Troubleshooting doc updates.

### PR DESCRIPTION
**Pull request by htroisi**
Status: Land after PX-127 is done.

### CODE REVIEW

File: content/en/01-about-pixie/04-faq.md [hunk 1]
```diff
@@ -193,13 +193,13 @@ This step of the deployment checks that the `vizier-cloud-connector` pod can suc
 
 *Deploy with CLI fails waiting for the Cloud Connector to come online.*
 
-This step of the deployment checks that the Cloud Connector can successfully communicate with Pixie Cloud. To debug this step, check the Pixie debug logs for the vizier-cloud-connector` pod, check the firewall, etc.
+This step of the deployment checks that the Cloud Connector can successfully communicate with Pixie Cloud. To debug this step, check the Pixie debug logs for the `vizier-cloud-connector` pod, check the firewall, etc.
 
 ### Why does my cluster show as unavailable / unhealthy in the Live UI?
 
-Confirm that all of the `pl` namespace pods are ready and available using `kubectl get pods -n pl`. Deploying Pixie usually takes anywhere between 5-7 minutes. Once Pixie is deployed, it can take a few minutes for the UI to show that the cluster is healthy.
+Confirm that all of the `pl` and `px-operator` namespace pods are ready and available using `px debug pods`. Deploying Pixie usually takes anywhere between 5-7 minutes. Once Pixie is deployed, it can take a few minutes for the UI to show that the cluster is healthy.
```
Comment (nserrino) [inline content/en/01-about-pixie/04-faq.md:200]: does this return the px-operator pods?

Comment (htroisi) [reply content/en/01-about-pixie/04-faq.md:200]: @philkuz - Just wanted to check that you are still planning to add the `px-operator` pods to `px debug pods`.

Comment (htroisi) [reply content/en/01-about-pixie/04-faq.md:200]: Looks like it didn't get added at the same time as fixing the `px collect-logs` command (PX-127), so I filed a separate issue (PX-131).

Comment (htroisi) [reply content/en/01-about-pixie/04-faq.md:200]: `px debug pods` now returns some px-operator pods (just the `vizier-operator`, not the `pixie-operator`).

**Review by nserrino — APPROVED** [review]

**Pull request closed without merging**